src/components/SubmissionConfirmation.js

```diff
@@ -140,6 +140,20 @@ const SubmissionConfirmation = ({statusUrl, onFailure, onConfirmed}) => {
 
   const showBackToMainWebsite = mainWebsiteUrl && !paymentUrl;
 
+  let linkTitle;
+
+  // Support FormattedMessage for backwards compatibility
+  try {
+    linkTitle = form.submissionReportDownloadLinkTitle;
+  } catch (error) {
+    linkTitle = (
+      <FormattedMessage
+        description="Download report PDF link title"
+        defaultMessage="Download PDF"
+      />
+    );
+  }
```

Comment:
I don't think this will throw an error, since `form` is an object, `linkTitle` will just resolve to `undefined` if the attribute is not present.